### PR TITLE
Dockerized

### PR DESCRIPTION
**Pull request by elgalu**

### CODE REVIEW

**Comment by mycargus** [comment]
I think my wildest dockerized dreams may come true when this pull request gets merged, but I want to confirm before I let myself get emotionally attached to it. 😄 

Is the goal here to allow using Zalenium in a fully dockerized configuration? Even with SauceLabs?

**Comment by elgalu** [comment]
> Allow using Zalenium in a fully dockerized configuration such that the only dependencies one would need on his or her host machine is Docker (no Java)

Indeed!

> my wildest dockerized dreams may come true when this pull request gets merged

You made my day with that comment :joy_cat: :smile: 

> the goal is to make this work with Sauce Labs

Yes!

Hey! I [see](https://github.com/mycargus/pact-poc) you seem to be using Pacts as [we](https://github.com/elgalu/pacts) :beers:

File: src/main/java/de/zalando/tip/zalenium/util/CommonProxyUtilities.java [hunk 1]
```diff
@@ -34,6 +35,29 @@ public static JsonElement readJSONFromUrl(String url) {
         return null;
     }
 
+    public static JsonElement readJSONFromFile(String fileName) {
+        try {
+            FileReader fr = new FileReader(new File(currentLocalPath(), fileName));
```
Comment (diemol) [inline src/main/java/de/zalando/tip/zalenium/util/CommonProxyUtilities.java]: BLOCKER Close this "FileReader". [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2095)

**Comment by diemol** [comment]
SonarQube analysis reported 1 issue
* BLOCKER 1 blocker

Watch the comments in this conversation to review them.

**Comment by diemol** [comment]
Getting closer, we are already building the image automatically and testing it with a few integration tests. 
We will figure out where to push the image soon, update the readme and release.

**Comment by diemol** [comment]
Commit 5161ee7 failed in Travis because SonarQube open source is down... 
I will run the build again in a few minutes.

**Pull request merged**